coord/src/config.h

```diff
@@ -37,6 +40,7 @@ extern SearchClusterConfig clusterConfig;
     .timeoutMS = 0,                                                                        \
     .globalPass = NULL,                                                                    \
     .cursorReplyThreshold = 1,                                                             \
+    .coordinatorPoolSize = COORDINATOR_POOL_DEFAULT_SIZE,                                  \
```

Comment:
Can it be tested? (sanity test that `_ft.config get/set` is working as expected after moving to coordinator configs)

Reply:
I can. I think it would be better to add tests for all the coordinator configurations when we add `FT.CONFIG` to cluster mode. 
Do you want me to add a temporary test for this configuration only?

Reply:
I agree. Yes, please add a test just for this config